### PR TITLE
Clear features

### PR DESCRIPTION
**Pull request by lgdacunh**

### CODE REVIEW

**Comment by lgdacunh** [comment]
@arthurshau, please take a look if you see a problem about moving the clear feature code, and about renaming the file.

**Comment by arthurshau** [comment]
I just noticed there is a typo in the function name, left a comment. While we're at it, do you mind also changing the name of the command itself in `plugins/ocp/ocp-nvme.h` to `clear-pcie-correctable-errors`? So that the command name matches the spec as well (which is the important part). Sorry, I should have brought this up when the PR was first merged, but I didn't catch it then.

Feel free to change the names in the Solidigm plugin too, so that it matches OCP if you'd like.

**Comment by lgdacunh** [comment]
> I just noticed there is a typo in the function name, left a comment. While we're at it, do you mind also changing the name of the command itself in `plugins/ocp/ocp-nvme.h` to `clear-pcie-correctable-errors`? So that the command name matches the spec (which is the important part). Sorry, I should have brought this up when these changes were first merged, but I didn't catch it then.
> 
> Feel free to change the names in the Solidigm plugin too, so that it matches OCP if you'd like.

@arthurshau, All requests done!

**Comment by lgdacunh** [comment]
@igaw , Can you take a look at his pull request?

File: plugins/ocp/ocp-nvme.h [hunk 1]
```diff
@@ -21,7 +21,7 @@ PLUGIN(NAME("ocp", "OCP cloud SSD extensions", NVME_VERSION),
 		ENTRY("internal-log", "Retrieve and save internal device telemetry log", ocp_telemetry_log)
 		ENTRY("clear-fw-activate-history", "Clear firmware update history log", clear_fw_update_history)
 		ENTRY("eol-plp-failure-mode", "Define EOL or PLP circuitry failure mode.", eol_plp_failure_mode)
-		ENTRY("clear-pcie-correctable-error-counters", "Clear PCIe correctable error counters", clear_pcie_corectable_error_counters)
+		ENTRY("clear-pcie-correctable-error", "Clear PCIe correctable error counters", clear_pcie_correctable_error_counters)
```
Comment (arthurshau) [inline plugins/ocp/ocp-nvme.h:24]: nit: Should be `ENTRY("clear-pcie-correctable-errors", ...` (missing an 's').

**Comment by arthurshau** [comment]
Just one tiny nit with the name of the OCP command, other than that, LGTM.

File: plugins/ocp/ocp-clear-features.c [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+}
+
+int ocp_clear_fw_update_history(int argc, char **argv, struct command *cmd, struct plugin *plugin)
+{
+	const char *desc = "OCP Clear Firmware Update History";
+
+	return ocp_clear_feature(argc, argv, desc, OCP_FID_CLEAR_FW_ACTIVATION_HISTORY);
+}
+
+int ocp_clear_pcie_corectable_error_counters(int argc, char **argv, struct command *cmd,
```
[the remark this answers is not in the document]
Comment (lgdacunh) [reply plugins/ocp/ocp-clear-features.c]: Good catch! Done!

**Comment by igaw** [comment]
Thanks!

**Pull request merged**